### DISCUSSION TITLE
Copy next code instead of current one when less than 3 seconds

### DISCUSSION THREAD

**Discussion by dannyhartono** [ente-io/ente#3159] [Enhancements]
Current behavior is copying the current code regardless of how many seconds are left. It would be great if tap-to-copy copies the next code instead when the time is less than 3 seconds.

**Comment by dnred** [comment]
You can click on the next code to copy it and the codes still work for a bit even after they've expired.

**Reply by dannyhartono** [reply]
Ah okay,, didn't know that
